jdaviz/app.py

```diff
@@ -311,16 +311,39 @@ def _link_new_data(self):
 
         # Can't link if there's no world_component_ids
         wc_new = self.data_collection[-1].world_component_ids
-        if wc_new == []:
-            return
+        pc_new = self.data_collection[-1].pixel_component_ids
+
+        # if wc_new == []:
+        #     return
```

Comment:
I'm not sure what the consequences are of commenting out this line are. I need to do it so that moment map data (which does not have `world_component_ids`) can be linked to the rest of the data in `data_collection`.